src/qml/README.md

```diff
@@ -7,7 +7,7 @@ This directory contains the source code for an experimental Bitcoin Core graphic
 Insecure CI artifacts are available for local testing of the master branch, avoiding the need to build:
 - for Windows: [`insecure_win_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip)
 - for macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
-- for Android: [`insecure_android_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip)
+- for ARM64 Android: [`insecure_android_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip)
```

Comment:
Is there a way to reconfigure the URL and the corresponding CI artifact name so that:
`insecure_android_apk.zip` is replaced by `insecure_android_arm64_apk.zip`.

```suggestion
- for ARM64 Android: [`insecure_android_arm64_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_arm64_apk.zip)
```

If possible, we should make this change, as this will make it more straightforward for the user to understand the artifact's intended target architecture.

Reply:
I think this is fine as is until we are to support x86 android